desk/app/expose.hoon

```diff
@@ -0,0 +1,2233 @@
+::  exposé: clearweb content rendering
+::
+::    copy a reference to a notebook post, then:
+::    :expose [%show /that/reference/with/id/numbers/quoted/'123456789']
+::    then visit in the browser:
+::    /expose/that/reference/as/copied/123456789
+::
+/-  c=cite, d=channels, co=contacts
+/+  sigil, u=channel-utils,
+    dbug, verb
+::
+|%
++$  state-0
+  $:  %0
+      open=(set cite:c)  ::TODO  could support ranges of msgs?
+  ==
+::
++$  action
+  $%  [%show =path]
+      [%hide =path]
+  ==
+::
++$  card  card:agent:gall
+::
+++  e
+  |%
+  ++  render-post
```

Comment:
we probably should move some of this out to other files, styles and scripts, or maybe all the rendering? not sure, but I think it makes this agent hard to read